sql/mysql-compatibility.md

```diff
@@ -31,6 +31,16 @@ However, in TiDB, the following MySQL features are not supported for the time be
 
 The auto-increment ID feature in TiDB is only guaranteed to be automatically incremental and unique but is not guaranteed to be allocated sequentially. Currently, TiDB is allocating IDs in batches. If data is inserted into multiple TiDB servers simultaneously, the allocated IDs are not sequential.
 
+> **Note**:
+> 
+> When multiple TiDB servers use the auto-increment ID, do not mix the default value and the custom value, because it reports an error in the following situation:
```

Comment:
When multiple TiDB servers use the auto-increment ID, -> If you use the auto-increment ID in a cluster with multiple TiDB servers,